### DISCUSSION TITLE
Any way to put an arrow in search box to start searching

### DISCUSSION THREAD

**Discussion by AndyIreland** [valinet/ExplorerPatcher#2858] [Q&A]
Hi, I cannot remember which version of windows, but windows 11 file explorer starts searching files and folders automatically as soon as you start writing into the search box. In a previous version of windows you could write into the search box in file explorer then press the arrow (or press enter)  to search. is there such a setting in explorer patcher or could one be introduced? 

![image](https://github.com/valinet/ExplorerPatcher/assets/49172217/cafd7522-31b9-433a-8883-830ed0337a2c)

**Comment by Amrsatrio** [comment]
That's the version with modern search box that no one liked. It was recently removed in Windows 10.

**Reply by AndyIreland** [reply]
I hate with the win 11 search box (in file explorer) that you start typing and it automatically starts searching then you have to keep clicking into the search box to finish off the word/file/folder you want it to search - it doesnt even wait to search until you have finished typing!
